### ISSUE TITLE
Releases > 1.3.0 are not pulled in by composer

### ISSUE THREAD

**Issue by sabl0r** [mike-kfed/roundcube-thunderbird_labels#44]
Any idea why newer releases like 1.3.2 are not pulled in?

Steps to reproduce:

```
foo /tmp $ git clone https://github.com/roundcube/roundcubemail.git
Cloning into 'roundcubemail'...
remote: Enumerating objects: 187, done.
remote: Counting objects: 100% (187/187), done.
remote: Compressing objects: 100% (129/129), done.
remote: Total 117773 (delta 67), reused 107 (delta 52), pack-reused 117586
Receiving objects: 100% (117773/117773), 57.13 MiB | 2.60 MiB/s, done.
Resolving deltas: 100% (81847/81847), done.
```

`foo /tmp $ cd roundcubemail/`

`foo /tmp/roundcubemail (master) $ cp composer.json-dist composer.json`

```
foo /tmp/roundcubemail (master) $ composer require weird-birds/thunderbird_labels
Using version ^1.3 for weird-birds/thunderbird_labels
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 42 installs, 0 updates, 0 removals
  - Installing composer/semver (1.4.2): Loading from cache
  - Installing roundcube/plugin-installer (0.1.9): Loading from cache
  - Installing pear/pear_exception (v1.0.0): Loading from cache
  - Installing pear/auth_sasl (v1.1.0): Loading from cache
  - Installing pear/net_idna2 (v0.2.0): Loading from cache
  - Installing pear/net_socket (v1.2.2): Loading from cache
  - Installing pear/console_getopt (v1.4.1): Loading from cache
  - Installing pear/pear-core-minimal (v1.10.7): Loading from cache
  - Installing pear/net_smtp (1.8.1): Loading from cache
  - Installing pear/console_commandline (v1.2.2): Loading from cache
  - Installing pear/crypt_gpg (v1.6.3): Loading from cache
  - Installing weird-birds/thunderbird_labels (v1.3.0): Loading from cache
```

**Comment by mike-kfed** [comment]
I don't use composer for anything, therefore I don't know how it works or what it does. If you find out what the problem is let me know and I change whatever is needed.

**Comment by alecpl** [comment]
This is handled by plugins.roundcube.net and plugin-installer. I see 1.3.1 and 1.3.2 on plugins.roundcube.net, but not 1.4rc2. Why? I don't. know. Maybe a bug that you could ask about on https://github.com/roundcube/plugin-installer/issues

**Comment by mike-kfed** [comment]
I still don't use composer. If you can find out what causes this and how to fix it, please do so and share it here. I'll implement what's needed when explained.

**Comment by skiner68** [comment]
Maybe... try to add missing version info to your composer.json, follow version constraints from docs https://getcomposer.org/doc/articles/versions.md

**Comment by mike-kfed** [comment]
Documentation says "The version of the package. In most cases this is not required and should be omitted" so the version info is not missing in my composer.json because it is not supposed to be there. Also I am using tags with their required versioning scheme in github, so that should be fine too.

**Comment by alecpl** [comment]
The source code for plugins.roundcube.net is at https://github.com/roundcube/plugins.roundcube.net

It is based on packagist and long behind packagist releases. Unfortunately update is not that easy. There's a ticket for this.

**Comment by mike-kfed** [comment]
alright I added version info and some other stuff I found in other plugins composer.json files. The documentation of plugins.roundcube also does not mention to put version info in the file btw, whatever this composer thing does is not well implemented/documented... I hope those changes help

**Comment by alecpl** [comment]
At the moment 1.4rc2 is there, but not 1.4.2. Maybe @thomascube would know something more?

**Comment by thomascube** [comment]
Maybe `v1.4rc2` is not considered as stable by Composer. Try to adjust the [minimum-stability](https://getcomposer.org/doc/04-schema.md#minimum-stability) setting in your `composer.json` file.

**Comment by mike-kfed** [comment]
1.4.2 is stable and newer than 1.4rc2, however the "current" version found by the plugins website is rc2. no clue why. I also don't know how this thing updates it's info because 1.4.2 is many days old, even a weekly scrape of the repositories should have found that by now.

**Comment by thomascube** [comment]
@mike-kfed You can trigger an update by logging in to plugins.roundcube.net and selecting your package. Then there's a button for it. However, I just triggered an update for your plugin manually.

**Comment by mike-kfed** [comment]
thanks, let's hope this resolves the issue

**Comment by mike-kfed** [comment]
seems to be fixed by the new packagist backend, just released v1.4.6 was immediately picked up by it. plugins.roundcube.net seems to lag a bit but that is fine I guess.
